packages/malloy/src/model/malloy_types.ts

```diff
@@ -1215,7 +1285,7 @@ export type QueryToMaterialize = {
 };
 
 export interface CompiledQuery extends DrillSource {
-  structs: StructDef[];
+  structs: SourceDef[];
```

Comment:
Aren't these `structs` actually `StructDef`s not `SourceDef`s?

Reply:
I counted on type checking to tell me when that was true.

Reply:
Yeah I don't know what this is

Reply:
If the whole PR goes green, AND I can make a vscode plugin with the green version of the source, I am going to ignore this and move on.